interfaces/builtin/shutdown.go

```diff
@@ -23,9 +23,8 @@ const shutdownSummary = `allows shutting down or rebooting the system`
 
 const shutdownBaseDeclarationSlots = `
   shutdown:
-    allow-installation:
-      slot-snap-type:
-        - core
+    allow-installation: false
+    deny-connection: true
     deny-auto-connection: true
 `
 
```

Comment:
this is not correct for an implicit interface like this one, see kernel_module_control.go for something appropriate for this

Reply:
so should I revert my changes on `shutdownBaseDeclarationSlots` and define `shutdownBaseDeclaration` with the following content?

```
const shutdownBaseDeclaration = `
  shutdown:
    allow-installation: false
    deny-auto-connection: true
`
```